frontend/src/slices/file/action.ts

```diff
@@ -0,0 +1,17 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { type FileUploadResponseDto } from 'shared/build/index.js';
```

Comment:
Maybe it should be imported from `types` folder?

Reply:
Thanks for having my back

- [65654a1](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/118/commits/65654a1286d0d5908f927dd8b03a652952aea9d5)